docs/topics_5/web-access-console.adoc

```diff
@@ -3,7 +3,7 @@
 [id='access_web_console_{context}']
 = Accessing the {WebNameTitle}
 
-Once started, the {WebName} is accessible from a browser by default on the local host at link:http://localhost:8080/mta-web[http://localhost:8080/mta-web].
+Once started, the {WebName} is accessible from a browser by default on the local host at link:``\http://localhost:8080/mta-web[\http://localhost:8080/mta-web`].
```

Comment:
Why are there 2 backticks before the escape character and an escape (\) before the URL text? You don't need to correct it because this file is obsolete but you need to understand how the markup works.